ompi/mca/coll/ucc/coll_ucc.h

```diff
@@ -0,0 +1,148 @@
+/**
+  Copyright (c) 2021      Mellanox Technologies. All rights reserved.
+  $COPYRIGHT$
+
+  Additional copyrights may follow
+
+  $HEADER$
+ */
+
+#ifndef MCA_COLL_UCC_H
+#define MCA_COLL_UCC_H
+
+#include "ompi_config.h"
+#include "mpi.h"
+#include "ompi/mca/mca.h"
+#include "opal/memoryhooks/memory.h"
+#include "opal/mca/memory/base/base.h"
+#include "ompi/mca/coll/coll.h"
+#include "ompi/communicator/communicator.h"
+#include "ompi/attribute/attribute.h"
+#include "ompi/op/op.h"
+#include "coll_ucc_debug.h"
+#include <ucc/api/ucc.h>
+
+BEGIN_C_DECLS
+
+#define COLL_UCC_CTS (UCC_COLL_TYPE_BARRIER   | UCC_COLL_TYPE_BCAST    | \
+                      UCC_COLL_TYPE_ALLREDUCE | UCC_COLL_TYPE_ALLTOALL | \
+                      UCC_COLL_TYPE_ALLTOALLV)
+
+#define COLL_UCC_CTS_STR "barrier,bcast,allreduce,alltoall,alltoallv"
```

Comment:
do we need to add non-blocking colls in this list? (or) name here represents both

Reply:
i will add separate names i think - lets be explicit.